packages/cli/test/agent-commands.test.js

```diff
@@ -0,0 +1,328 @@
+            const command = 'bun';
+            const args = ['start', '--character', characterFilePath];
+
+            child = spawn(command, args, {
+                cwd: projectRoot,
+                stdio: ['ignore', 'pipe', 'pipe'],
+                detached: true // Allows proper process tree management
+            });
+
+            // Create promise to wait for expected output
+            const outputPromise = new Promise((resolve, reject) => {
+                let stdoutData = '';
+
+                child.stdout.on('data', (data) => {
+                    const output = data.toString();
+                    stdoutData += output;
+                    console.log(output);
+                    if (stdoutData.includes(`Successfully loaded character from : ${characterFilePath}`)) {
+                        resolve(stdoutData);
+                    }
+                });
+
+                child.on('error', (error) => {
+                    console.error('Child process error:', error);
+                    reject(error);
+                });
+
+                child.stderr.on('data', (data) => {
+                    console.error('Stderr:', data.toString()); // Log all stderr data
+                });
+            });
+
+
+            // Set timeout with cleanup
+            const timeoutPromise = new Promise((_, reject) => {
+                setTimeout(() => {
+                    if (child) {
+                        process.kill(-child.pid, 'SIGKILL'); // Kill entire process group
+                    }
+                    reject(new Error('Test timed out after 15 seconds'));
+                }, 15000);
+            });
+
+            // Wait for output or timeout
+            const stdout = await Promise.race([outputPromise, timeoutPromise]);
+            console.log(stdout);
+
+            // Assert
+            expect(stdout).toContain(`Successfully loaded character from: ${characterFilePath}`);
+            expect(stdout).toContain(agentName);
+        } finally {
+            // Cleanup - kill entire process tree
+            if (child) {
+                try {
+                    process.kill(-child.pid, 'SIGKILL');
+                } catch (e) {
+                    // Process might already be dead
+                }
+            }
+        }
+    }, 30000);
+
+
+
+    it('Start Agent - Start Multiple Agents', async () => {
+        // Arrange
+        const agent1Name = 'test-agent1';
+        const agent2Name = 'test-agent2';
+        const characters = [
+            {
+                name: agent1Name,
+                system: "You are a test agent 1.",
+                bio: ["A test agent for integration testing."],
+                plugins: []
+            },
+            {
+                name: agent2Name,
+                system: "You are a test agent 2.",
+                bio: ["A test agent for integration testing."],
+                plugins: []
+            }
+        ];
```

Comment:
let's move this to the constants file and reuse across test cases where applicable